test/functional/fixtures/run-options/allow-multiple-windows/test.js

```diff
@@ -0,0 +1,36 @@
+const { expect } = require('chai');
+
+describe('Allow multiple windows', () => {
+    describe('Switch to the opened window', () => {
+        it('Click on link', () => {
+            return runTests('testcafe-fixtures/switching-to-child/click-on-link.js', null, { allowMultipleWindows: true });
+        });
+
+        it('Form submit', () => {
```

Comment:
This test passes even with `{ allowMultipleWindows: false }`, because the first page contains `input` type = 'text' too